plugin-server/src/cdp/hog-executor.ts

```diff
@@ -99,11 +103,14 @@ export class HogExecutor {
                     }
                 }
             } catch (error) {
-                // TODO: This should be reported as a log or metric
-                status.error('🦔', `[HogExecutor] Error filtering function`, {
-                    hogFunctionId: hogFunction.id,
-                    hogFunctionName: hogFunction.name,
-                    error: error.message,
+                logs.push({
```

Comment:
I actually think we should consider these "errors" rather than being "filtered" out 🤔 now that i think about it...
But thats an additional change so happy for that to come later